code/modules/maps/overmap/planets/virgo2.dm

```diff
@@ -1,16 +1,14 @@
 // V2 doesnt actually have its own weather so this is gonna be here for now
 /datum/atmosphere/planet/virgo2
-	base_gases = list(
-	/datum/gas/nitrogen = 0.10,
-	/datum/gas/oxygen = 0.03,
-	/datum/gas/carbon_dioxide = 0.87
+	base = list(
+		GAS_ID_NITROGEN = 0.10,
+		GAS_ID_OXYGEN = 0.03,
```

Comment:
why are gases switched to gas_id here? either one is wrong

Reply:
fixed. atmosphere allows both but we generally want paths yeah.